src/EnergyPlus/OutputProcessor.cc

```diff
@@ -5212,7 +5212,7 @@ void SetupOutputVariable(EnergyPlusData &state,
         auto &thisRvar = op->RVariableTypes(CV);
         thisRvar.timeStepType = TimeStepType;
         thisRvar.storeType = VariableType;
-        thisRvar.VarName = KeyedValue + ':' + VarName;
+        thisRvar.VarName = fmt::format("{}:{}", KeyedValue, VarName);
```

Comment:
So here we actually store a concrete string, which is obviously necessary because the string_views may be temporaries in the functions calling this.  :+1: